contracts/ArchetypeLogic.sol

```diff
@@ -144,19 +144,11 @@ library ArchetypeLogic {
   ) public view returns (uint256) {
     uint256 price = invite.price;
     if (invite.interval != 0) {
-      uint256 diff = (((block.timestamp - invite.start) / invite.interval) * invite.delta);
+      uint256 diff = ((block.timestamp - invite.start) / invite.interval) * invite.delta;
       if (price > invite.reservePrice) {
-        if (diff > price - invite.reservePrice) {
-          price = invite.reservePrice;
-        } else {
-          price = price - diff;
-        }
+        price = diff > price - invite.reservePrice ? invite.reservePrice : price - diff;
       } else if (price < invite.reservePrice) {
-        if (diff > invite.reservePrice - price) {
-          price = invite.reservePrice;
-        } else {
-          price = price + diff;
-        }
+        price = diff > invite.reservePrice - price ? invite.reservePrice : price + diff;
```

Comment:
Are ternarys more efficient than full if syntax?

Reply:
They can be. I think it's VERY VERY marginal though. Probably talking 10-20 gas units. I just like the look of ternary expressions sometimes.